src/main/kotlin/com/personio/synthetics/builder/SyntheticBrowserTestBuilder.kt

```diff
@@ -64,6 +65,26 @@ class SyntheticBrowserTestBuilder(
         options.tickEvery = frequency.inWholeSeconds
     }
 
+    /**
+     * Sets the browsers and devices for the synthetic browser test
+     * @param deviceIds Pass comma separated browsers and devices to the test config derived from SyntheticsDeviceID class
+     */
+    fun browsersAndDevices(vararg deviceIds: SyntheticsDeviceID) {
+        options.deviceIds = deviceIds.map { it }
+    }
+
+    /**
+     * Sets the browsers and devices for the synthetic browser test
+     * @param deviceIds Pass comma separated browsers and devices to the test config derived from SyntheticsDeviceID class
+     */
+    @Deprecated(
+        message = "The function is deprecated. Please use `browsersAndDevices` instead.",
+        replaceWith = ReplaceWith("browsersAndDevices(*deviceIds)")
+    )
+    fun browserAndDevice(vararg deviceIds: SyntheticsDeviceID) {
```

Comment:
Why are we adding this deprecated function here? I find it a bit confusing that we'll have the new function `browsersAndDevices` above, along with this one, plus the old `browserAndDevice` function in the [BrowserTestConfig.kt](https://github.com/personio/datadog-synthetic-test-support/blob/master/src/main/kotlin/com/personio/synthetics/config/BrowserTestConfig.kt#L163) at the same time 🤔